.github/workflows/mobilecoin-workflow-dev-setup-environment.yaml

```diff
@@ -0,0 +1,235 @@
+      run: |
+        # Create seeds dir.
+        mkdir -p "${SEEDS_BASE_PATH}"
+
+        # Write values to be used as k8s secret values.
+        echo -n "${{ secrets.INITIAL_KEYS_SEED }}" > "${SEEDS_BASE_PATH}/INITIAL_KEYS_SEED"
+        echo -n "${{ secrets.FOG_KEYS_SEED }}" > "${SEEDS_BASE_PATH}/FOG_KEYS_SEED"
+        echo -n "${{ secrets.MNEMONIC_KEYS_SEED }}" > "${SEEDS_BASE_PATH}/MNEMONIC_KEYS_SEED"
+        echo -n "${{ secrets.MNEMONIC_FOG_KEYS_SEED }}" > "${SEEDS_BASE_PATH}/MNEMONIC_FOG_KEYS_SEED"
+        echo -n "${{ secrets.FOG_REPORT_SIGNING_CA_CERT }}" > "${SEEDS_BASE_PATH}/FOG_REPORT_SIGNING_CA_CERT"
+        echo -n "/wallet-seeds/FOG_REPORT_SIGNING_CA_CERT" > "${SEEDS_BASE_PATH}/FOG_REPORT_SIGNING_CA_CERT_PATH"
+        echo -n "fog://fog.${{ inputs.namespace }}.development.mobilecoin.com:443" > "${SEEDS_BASE_PATH}/FOG_REPORT_URL"
+
+    - name: Create wallet key secrets
+      uses: mobilecoinofficial/gha-k8s-toolbox@v1
+      with:
+        action: secrets-create-from-file
+        namespace: ${{ inputs.namespace }}
+        rancher_cluster: ${{ secrets.RANCHER_CLUSTER }}
+        rancher_url: ${{ secrets.RANCHER_URL }}
+        rancher_token: ${{ secrets.RANCHER_TOKEN }}
+        object_name: sample-keys-seeds
+        src: ${{ env.SEEDS_BASE_PATH }}
+
+    - name: Write minting keys
+      run: |
+        # Create minting secrets dir
+        mkdir -p "${MINTING_BASE_PATH}"
+
+        # Write values to be used as k8s secret values.
+        echo -n "${{ secrets.MINTING_1_GOVERNOR_1_PRIVATE }}" > "${MINTING_BASE_PATH}/token_1_governor_1.private.pem"
+        echo -n "${{ secrets.MINTING_1_GOVERNOR_1_PUBLIC }}" > "${MINTING_BASE_PATH}/token_1_governor_1.public.pem"
+        echo -n "${{ secrets.MINTING_1_SIGNER_1_PRIVATE }}" > "${MINTING_BASE_PATH}/token_1_signer_1.private.pem"
+        echo -n "${{ secrets.MINTING_1_SIGNER_1_PUBLIC }}" > "${MINTING_BASE_PATH}/token_1_signer_1.public.pem"
+        echo -n "${{ secrets.MINTING_8192_GOVERNOR_1_PRIVATE }}" > "${MINTING_BASE_PATH}/token_8192_governor_1.private.pem"
+        echo -n "${{ secrets.MINTING_8192_GOVERNOR_1_PUBLIC }}" > "${MINTING_BASE_PATH}/token_8192_governor_1.public.pem"
+        echo -n "${{ secrets.MINTING_8192_SIGNER_1_PRIVATE }}" > "${MINTING_BASE_PATH}/token_8192_signer_1.private.pem"
+        echo -n "${{ secrets.MINTING_8192_SIGNER_1_PUBLIC }}" > "${MINTING_BASE_PATH}/token_8192_signer_1.public.pem"
+
+        # values for v3.0.0-dev release
+        echo -n "${{ secrets.MINTING_8192_GOVERNOR_1_PRIVATE }}" > "${MINTING_BASE_PATH}/minter8192_governor.private.pem"
+        echo -n "${{ secrets.MINTING_8192_GOVERNOR_1_PUBLIC }}" > "${MINTING_BASE_PATH}/minter8192_governor.public.pem"
+        echo -n "${{ secrets.MINTING_8192_SIGNER_1_PRIVATE }}" > "${MINTING_BASE_PATH}/token8192_signer.private.pem"
+        echo -n "${{ secrets.MINTING_8192_SIGNER_1_PUBLIC }}" > "${MINTING_BASE_PATH}/token8192_signer.public.pem"
```

Comment:
:question: These look to be the same values as 5 linus up why write to another file instead of re-using?

Reply:
This is for compatibility with the previous release. 

The testing scripts are bundled in the release its self. That makes them internally consistent for a specific stage of the release, but it does mean that we need to make sure we keep compatible configs for as long as we are using the older release as part of the CD upgrade process.